### ISSUE TITLE
got training loss (per epoch) -4.47 at step 40k, and loss is decreasing very slow now, is this normal? or should I decrease my learning rate, which is 5e-5 with batch size of 4 

### ISSUE THREAD

**Issue by wenyong-h** [NVIDIA/waveglow#29]
Can you and everyone else share your loss curve for reference?

**Comment by wenyong-h** [comment]
I can get pretty good sample (good intelligibility with some sharp noise in the backgroud) at step 68k with loss -4.647.  So I guess things are OK.

**Comment by rafaelvalle** [comment]
A good guideline for decreasing the learning rate is when the loss curve plateaus.

**Comment by hdmjdp** [comment]
@wenyong-h what is your batchsize?

**Comment by wenyong-h** [comment]
@hdmjdp my batch size is 4.  I'm sorry I can't post the sample,  it's private data

**Comment by hdmjdp** [comment]
@wenyong-h can your share your config json, such as "seqlen". and your batch_size=4 is on single gpu, total batch_size=4*num_gpus?

**Comment by jiqizaisikao** [comment]
Hi,the max p=1 and  min(-log(p))=0 so, why  the loss<0

**Comment by rafaelvalle** [comment]
@jiqizaisikao take a look at https://github.com/NVIDIA/waveglow/issues/1
